tools/FakeItEasy.Build/Program.cs

```diff
@@ -57,7 +57,9 @@
     forEach: projectsToPack,
     action: project => Run("dotnet", $"pack {project.Path} --configuration Release --no-build --nologo --output \"{Path.GetFullPath("artifacts/output")}\""));
 
-Target("docs", DependsOn("check-docs-links", "check-readme-links"));
+Target("docs", DependsOn("generate-docs"));

```

Comment:
I worried that we'd have failures if we broke internal links (between pages), and would've really liked that to be checked immediately.
I tried introducing a typo just now and mkdocs was having none of it. Success!